### PR TITLE
Removed leading tabs and trailing spaces from ui/scripts/*.js

### PR DESCRIPTION
**Pull request by kevindierkx**

### CODE REVIEW

**Comment by borisroman** [comment]
@karuturi @DaanHoogland  @remibergsma As discussed on Slack August 3rd.

**Comment by wido** [comment]
LGTM

File: ui/scripts/instances.js [hunk 1]
```diff
@@ -1959,16 +1959,16 @@
...
-								isoid: {
+                                isoid: {
                   label: 'label.attached.iso',
                   isEditable: false,
                   converter: function(isoid) {
                     return cloudStack.converters.toBooleanText(isoid != null);
                   }
                 },
-								*/
+                                */
```
Comment (DaanHoogland) [inline ui/scripts/instances.js]: Let's remove these lines, and not reformat to keep comment in code

File: ui/scripts/regions.js [hunk 2]
```diff
@@ -197,7 +197,7 @@
                                     var region = args.context.regions[0];
 
                                     /* e.g.
-                  region.endpoint	== "http://localhost:8080/client/"
+                  region.endpoint    == "http://localhost:8080/client/"
                   document.location.href == "http://localhost:8080/client/#"
                   */
                                     /*
```
Comment (DaanHoogland) [inline ui/scripts/regions.js]: let's not reformat biut remove code in comment

File: ui/scripts/storage.js [hunk 3]
```diff
@@ -55,13 +55,13 @@
...
-							indicator: {
+                        state: {
+                          label: 'State',
+                            indicator: {
                 'Ready': 'on'
               }
-						}
-						*/
+                        }
+                        */
```
Comment (DaanHoogland) [inline ui/scripts/storage.js]: let's get rid of this bit of confusing code in comment

**Comment by borisroman** [comment]
We've used the following scripts to remove the trailing white spaces and leading tabs:


Tabs to 4 white spaces:
find . -iname '*.js' -type f -exec sed -i '' 's/\t/    /' {} \+

Remove trailing white spaces:
find . -iname '*.js' -type f -exec sed -i '' 's/[[:space:]]\{1,\}$//' {} \+

File: ui/scripts/templates.js [hunk 4]
```diff
@@ -519,16 +519,16 @@
...
+                                //XenServer only (starts here)
                                 if (args.$form.find('.form-item[rel=xenserverToolsVersion61plus]').css("display") != "none") {
                                     $.extend(data, {
                                         'details[0].hypervisortoolsversion': (args.data.xenserverToolsVersion61plus == "on") ? "xenserver61" : "xenserver56"
                                     });
                                 }
                                 //XenServer only (ends here)
-                                
-                                
+
```
Comment (DaanHoogland) [inline ui/scripts/templates.js]: why adding two empty lines here?

File: ui/scripts/templates.js [hunk 5]
```diff
@@ -2689,14 +2689,14 @@
...
             allowedActions.push("copyTemplate");
             /*
             if(g_regionsecondaryenabled != true) {
                 allowedActions.push("copyTemplate");
             }
             */
-			
+
             //allowedActions.push("createVm"); // For Beta2, this simply doesn't work without a network.
```
Comment (DaanHoogland) [inline ui/scripts/templates.js]: another bit to remove (the 7 lines above)

File: ui/scripts/templates.js [hunk 6]
```diff
@@ -2735,8 +2735,8 @@
             /*
             if(g_regionsecondaryenabled != true) {
                 allowedActions.push("copyISO");
-			}
-			*/
+            }
+            */
         }
```
Comment (DaanHoogland) [inline ui/scripts/templates.js]: eeeeeh, reformatted code in comment :catholiccrosssign: ;) let's remove

File: ui/scripts/ui-custom/regions.js [hunk 7]
```diff
@@ -37,9 +37,9 @@
                             $li.data('region-data', region);
 
                             /* e.g.
-							region.endpoint	== "http://localhost:8080/client/"
-							document.location.href == "http://localhost:8080/client/#"
-							*/
+                            region.endpoint    == "http://localhost:8080/client/"
+                            document.location.href == "http://localhost:8080/client/#"
+                            */
```
Comment (DaanHoogland) [inline ui/scripts/ui-custom/regions.js]: comment

File: ui/scripts/ui/widgets/detailView.js [hunk 8]
```diff
@@ -1062,19 +1062,19 @@
...
+                 var intervalKey = setInterval(function() {
+                 var toClearInterval = value.pollAgainFn(context);
+                 if(toClearInterval == true) {
+                 clearInterval(intervalKey);
+                 $('.detail-view .toolbar .button.refresh').click();     //click Refresh button to refresh detailView
+                 }
+                 }, 2000);
+                 }
+                 }
+                 */
```
Comment (DaanHoogland) [inline ui/scripts/ui/widgets/detailView.js]: exorcise this bit!

File: ui/scripts/ui/widgets/listView.js [hunk 9]
```diff
@@ -275,7 +275,7 @@
                                                         });
                                                     } else {
                                                         $newRow = replaceItem($instanceRow,
-                                                    	    args.data, //$.extend($instanceRow.data('json-obj'), args.data), /* $.extend($instanceRow.data('json-obj'), args.data) causes CLOUDSTACK-4687 */
+                                                            args.data, //$.extend($instanceRow.data('json-obj'), args.data), /* $.extend($instanceRow.data('json-obj'), args.data) causes CLOUDSTACK-4687 */
```
Comment (DaanHoogland) [inline ui/scripts/ui/widgets/listView.js]: once someone understands this comment, please replace it with something

File: ui/scripts/vpc.js [hunk 10]
```diff
@@ -155,41 +155,41 @@
...
-                    for (var vmIndex = 0; vmIndex < selectedVMs.length; vmIndex++) {      
+                    for (var vmIndex = 0; vmIndex < selectedVMs.length; vmIndex++) {
                         var selectedIPs = selectedVMs[vmIndex]._subselect;
                         for (var ipIndex = 0; ipIndex < selectedIPs.length; ipIndex++) {
                             inputData['vmidipmap[' + vmidipmapIndex + '].vmid'] = selectedVMs[vmIndex].id;
-                            
-                            //"ipAddresses" is not in args.context since this LB rule is under a VPC, not an address. 
+
+                            //"ipAddresses" is not in args.context since this LB rule is under a VPC, not an address.
                             /*
```
Comment (DaanHoogland) [inline ui/scripts/vpc.js]: yet another useless bit reformatted

File: ui/scripts/zoneWizard.js [hunk 11]
```diff
@@ -963,16 +963,16 @@
                                 if ($(this).val() == "VMware") {
                                     //$('li[input_sub_group="external"]', $dialogAddCluster).show();
                                     if (dvSwitchEnabled) {
-                                        /*   
-										$fields.filter('[rel=vSwitchPublicType]').css('display', 'inline-block');
-										$form.find('.form-item[rel=vSwitchGuestType]').css('display', 'inline-block');
+                                        /*
```
Comment (DaanHoogland) [inline ui/scripts/zoneWizard.js]: wow, more code that won't run

File: ui/scripts/zoneWizard.js [hunk 12]
```diff
@@ -983,12 +983,12 @@
 
 
                                     } else {
-                                        /*    
-										$form.find('.form-item[rel=vSwitchPublicType]').css('display', 'none');
-									    $form.find('.form-item[rel=vSwitchGuestType]').css('display', 'none');
-									    $form.find('.form-item[rel=vSwitchPublicName]').css('display','none');
-									    $form.find('.form-item[rel=vSwitchGuestName]').css('display','none');
-						                */
+                                        /*
```
Comment (DaanHoogland) [inline ui/scripts/zoneWizard.js]: more comments

File: ui/scripts/zoneWizard.js [hunk 13]
```diff
@@ -1139,15 +1139,15 @@
                         dependsOn: 'overridepublictraffic'
                     },
                     */
```
Comment (DaanHoogland) [inline ui/scripts/zoneWizard.js]: a not reformatted bit but remove makes sense as well here

File: ui/scripts/zoneWizard.js [hunk 14]
```diff
@@ -1139,15 +1139,15 @@
...
-					
-					/*
+
+                    /*
                     vSwitchPublicName: {
                         label: 'Public Traffic vSwitch Name',
                         dependsOn: 'overridepublictraffic',
                         isHidden: true
                     },
                     */
```
Comment (DaanHoogland) [inline ui/scripts/zoneWizard.js]: let's remove

File: ui/scripts/zoneWizard.js [hunk 15]
```diff
@@ -1221,14 +1221,14 @@
 
                     },
                     */
```
Comment (DaanHoogland) [inline ui/scripts/zoneWizard.js]: remove

File: ui/scripts/zoneWizard.js [hunk 16]
```diff
@@ -1221,14 +1221,14 @@
...
-					/*
+
+                    /*
                     vSwitchGuestName: {
                         label: 'Guest Traffic vSwitch Name',
                         dependsOn: 'overrideguesttraffic',
                         isHidden: true
                     },
-					*/
+                    */
```
Comment (DaanHoogland) [inline ui/scripts/zoneWizard.js]: let's remove

File: ui/scripts/zoneWizard.js [hunk 17]
```diff
@@ -1972,45 +1972,45 @@
...
+                                success: function(json) {
+                                    var s3stores = json.listimagestoresresponse.imagestore;
+                                    if(s3stores != null && s3stores.length > 0) {
+                                        storageproviders.push({ id: 'S3', description: 'S3'}); //if (region-wide) S3 store exists already, only "S3" option should be included here. Any other type of store is not allowed to be created since cloudstack doesn't support multiple types of store at this point.
+                                    } else {
+                                        /*
+                                        UI no longer gets providers from "listStorageProviders&type=image" because:
+                                        (1) Not all of returned values are handled by UI (e.g. Provider "NetApp" is not handled by UI).
+                                        (2) Provider "SMB" which is handled by UI is not returned from "listStorageProviders&type=image"
+                                        */
```
Comment (DaanHoogland) [inline ui/scripts/zoneWizard.js]: remove

File: ui/scripts/zoneWizard.js [hunk 18]
```diff
@@ -2108,17 +2108,17 @@
...
-                                        	}                                        	
-                                        	
-                                        	//NFS
+                                            $fields.filter('[rel=name]').css('display', 'inline-block');
+
+                                            if(s3stores != null && s3stores.length > 0) {
+                                                $fields.filter('[rel=name]').find('input').val(s3stores[0].name);
+                                                $fields.filter('[rel=name]').find('input').attr("disabled", "disabled");
+                                            } else {
+                                                //$fields.filter('[rel=name]').find('input').val("");
```
Comment (DaanHoogland) [inline ui/scripts/zoneWizard.js]: remove

File: ui/scripts/zoneWizard.js [hunk 19]
```diff
@@ -4363,12 +4363,12 @@
                         array1.push("&password=" + todb(args.data.cluster.vCenterPassword));
 
                         //dvswitch is enabled
-						/*
+                        /*
                         if (args.data.cluster.vSwitchPublicType != "")
                             array1.push('&publicvswitchtype=' + args.data.cluster.vSwitchPublicType);
-						*/
+                        */
```
Comment (DaanHoogland) [inline ui/scripts/zoneWizard.js]: remove

File: ui/scripts/zoneWizard.js [hunk 20]
```diff
@@ -4363,12 +4363,12 @@
...
 
-						/*
+                        /*
                         if (args.data.cluster.vSwitchPublicName != "")
                             array1.push("&publicvswitchname=" + args.data.cluster.vSwitchPublicName);
                         */
```
Comment (DaanHoogland) [inline ui/scripts/zoneWizard.js]: remove

File: ui/scripts/zoneWizard.js [hunk 21]
```diff
@@ -4377,23 +4377,23 @@
...
                             array1.push('&guestvswitchtype=' + args.data.cluster.vSwitchGuestType);
                         */
-						
-						/*
+
+                        /*
                         if (args.data.cluster.vSwitchGuestName != "")
                             array1.push("&guestvswitchname=" + args.data.cluster.vSwitchGuestName);
-						*/
+                        */
```
Comment (DaanHoogland) [inline ui/scripts/zoneWizard.js]: remove

File: ui/scripts/templates.js [hunk 22]
```diff
@@ -2746,7 +2722,7 @@
      if (((isAdmin() == false && !(jsonObj.domainid == g_domainid && jsonObj.account == g_account))  //if neither root-admin, nor item owner
      || jsonObj.isready == false)
      || (jsonObj.bootable == false)
-     || (jsonObj.domainid ==	1 && jsonObj.account ==	"system")
+     || (jsonObj.domainid ==    1 && jsonObj.account ==    "system")
```
Comment (DaanHoogland) [inline ui/scripts/templates.js]: please remove this block of commented code

File: ui/scripts/ui-custom/accountsWizard.js [hunk 23]
```diff
@@ -130,89 +130,89 @@
...
+                                                "email": "test@test.com",
+                                                "principal": "CN=test1dddd,CN=Users,DC=hyd-qa,DC=com",
+                                                "firstname": "test1",
+                                                "username": "test1dddd",
+                                                "domain": "CN=test1dddd"
+                                            }
+                                        ]
+                                    }
+                                };
+                            */
```
Comment (DaanHoogland) [inline ui/scripts/ui-custom/accountsWizard.js]: remove

**Comment by kevindierkx** [comment]
I've removed the above mentions of commented code. Should be all set.

**Comment by DaanHoogland** [comment]
LGTM, i will have a go at merging when I have Miguels new version of the merge script and the origin/apache triangle ready on my new (post schuberg philis) workstation

**Pull request merged**